frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -0,0 +1,180 @@
+import { getIn, useFormikContext } from 'formik';
+import { groupBy, sortBy } from 'lodash';
+import React, { Fragment, useEffect, useState } from 'react';
+import { Highlighter, Menu, MenuItem } from 'react-bootstrap-typeahead';
+import { Label } from '../Label';
+import { SelectOption, SelectOptions } from './Select';
+import { TypeaheadField } from './Typeahead';
+
+interface IParentSelect {
+  /** specify the field that is being accessed */
+  field: string;
+  /** give the component the desired options to group by parent with */
+  options: SelectOptions;
+  /** if you would like placeholder text within the input */
+  placeholder?: string;
+  /** filterBy is optional - if used can determine what you can filter with */
+  filterBy?: string[];
+  /** provide the opitional label text*/
+  label?: string;
+  /** flag used to control whehter to display red astrix indicating required */
+  required?: boolean;
+  /** determine whether this component is disabled or not */
+  disabled?: boolean;
+  /** if this component is being used as a filter do not pre populate the value */
+  isFilter?: boolean;
```

Comment:
I've always found this property naming confusing.  This is really about whether the component has a default value I think.